tools/MakeAppxHelper.cmd

```diff
@@ -29,15 +29,14 @@ if "%TFS_BUILDCONFIGURATION%" EQU "" (
 
 set BasePackageName=Microsoft.ProjectReunion
 
-echo BUILDOUTPUT_OVERRIDE = %BUILDOUTPUT_OVERRIDE%
-if "%BUILDOUTPUT_OVERRIDE%" == "" (
-	set InputDirectory=%CD%\..\BuildOutput\%TFS_BUILDCONFIGURATION%\%TFS_PLATFORM%\ProjectReunion_dll
-	set OutputDirectory=%CD%\..\BuildOutput\%TFS_BUILDCONFIGURATION%\%TFS_PLATFORM%\FrameworkPackage
-) else (
-	set InputDirectory=%BUILDOUTPUT_OVERRIDE%\ProjectReunion_dll
-	set OutputDirectory=%BUILDOUTPUT_OVERRIDE%\FrameworkPackage
+if "%BUILDOUTPUT_OVERRIDE%" EQU "" (
```

Comment:
Can this happen? How come we don't set it anymore if it does happen?

Reply:
It doesn't happen anywhere in our pipeline.  The only place we were calling this, we set the environment variable immediately beforehand, so I just got rid of the default option to ensure that still happened.

Reply:
Just seems like a weird check if we know we are setting it immediately beforehand.  I guess exiting gracefully is a better option than an exception.